_docs/templates/_npe2_writers_guide.md.jinja

```diff
@@ -31,7 +31,7 @@ by the corresponding writer contribution in the manifest).
    **attributes**, (e.g. `{'name': 'My Layer', 'opacity': 0.6}`)
 
    ```python
-   def single_layer_writer(path: str, data: Any, attributes: dict) -> Optional[str]:
+   def single_layer_writer(path: str, data: Any, attributes: dict) -> List[str]:
```

Comment:
Based on the formal type definition below, it seems like this should be `List[str]` instead of `Optional[str]`. In practice, I tried returning `None`, `path`, and `[path]` and they all worked, so may not be too important.